src/main/java/com/faforever/client/teammatchmaking/TeamMatchmakingService.java

```diff
@@ -298,14 +298,14 @@ public void onPartyInvite(PartyInvite message) {
   }
 
   @VisibleForTesting
-  protected void acceptPartyInvite(PlayerBean player) {
+  void acceptPartyInvite(PlayerBean player) {
     if (isCurrentlyInQueue()) {
-      notificationService.addNotification(new ImmediateNotification(
-          i18n.get("teammatchmaking.notification.joinAlreadyInQueue.title"),
-          i18n.get("teammatchmaking.notification.joinAlreadyInQueue.message"),
-          Severity.WARN,
-          Collections.singletonList(new Action(i18n.get("dismiss")))
-      ));
+      notificationService.addImmediateWarnNotification("teammatchmaking.notification.joinAlreadyInQueue.message");
+      return;
```

Comment:
The same refactoring could be applied for invitePlayer() below

Reply:
You don't really need to have the same check when inviting a player. Because when you invite a player you are in control of when you queue and you can't queue while a game is running.

Reply:
No, I meant only the refactoring from `addNotification` to `addImmediateWarnNotification`